.travis.yml

```diff
@@ -1,3 +1,4 @@
+dist: xenial
```

Comment:
Just confirming this is not different as of recently because of this change right: https://blog.travis-ci.com/2019-04-15-xenial-default-build-environment

Reply:
@jbabichjapan Good point, it seems their work is been delayed:
https://changelog.travis-ci.com/xenial-as-the-default-build-environment-99476

I mainly needed to specify this to let the Travis use the latest PyPy version (`PyPy: 7.x`). Otherwise, they use old versions as we listed here ( `PyPy: 2.x, 4.x` ):
https://github.com/twitterdev/twitter-python-ads-sdk#compatibility--versioning

You can see the difference below:
_without_ `dist:` key: https://travis-ci.org/twitterdev/twitter-python-ads-sdk/jobs/537644496
> $ python --version
> Python 3.5.3 (a39af0be3a22, Jun 05 2017, 20:18:00)
> [PyPy 5.8.0-beta0 with GCC 6.2.0 20160901]

_with_ `dist:` key: https://travis-ci.org/twitterdev/twitter-python-ads-sdk/jobs/538559585
> $ python --version
> Python 3.6.1 (784b254d6699, Apr 14 2019, 10:22:42)
> [PyPy 7.1.1-beta0 with GCC 6.2.0 20160901]